### ISSUE TITLE
Can't activate PRO on previous version of extension

### ISSUE THREAD

**Issue by greqov** [wallabyjs/quokka#934]
### Issue description or question

Is this issue related to Quokka not outputting the expected results of your code?: No

My license expired and now I can't activate PRO version. The [site](https://quokkajs.com/docs/previous.html?expirydate=2023-5-4#previous-versions) tells me: 
```
Your license expired on 4 May, 2023
The last version of Quokka that was included in your 12-months of free upgrades was: Quokka-1.0.534 for VS Code
``` 
So I've installed that 1.0.534 version, activated license (via license key) on extension page. But if I try to run quokka at js file (via "Start on current file" command), editor shows me a notification about upgrading
```
Your Quokka.js 'Pro' license free upgrades period has expired. If you would like to work with the latest version of Quokka.js 'Pro' and future versions released within the next 12 months, please renew your license. If you have already purchased the new license, please activate it.
```

### Quokka.js Console Output

```
Cannot find module './quokka.js' imported from /home/ej/.vscode/extensions/wallabyjs.quokka-vscode-1.0.534/dist/wallaby/runners/node/quokka@1.0.0/initializer.js 
```

Am I doing something wrong? I thought I can use old purchased version "forever".

### Code editor version

Visual Studio Code v1.86.1
Node.js: 18.17.1

### OS name and version

OS: Linux x64 6.7.4-arch1-1

**Comment by smcenlly** [comment]
> I thought I can use old purchased version "forever".

Yes - you can.

It sounds like the message you are receiving is simply a warning that your license has expired, not that there is a problem. You may disable the notification with the VS Code setting `quokka.suppressExpirationNotifications`.

The `Cannot find module './quokka.js' imported from quokka@1.0.0/initializer.js` may be related to breaking changes in node.js that are not supported by older versions of Quokka.

---

Can you please try the `quokka.suppressExpirationNotifications` setting and see if the warning disappears?

Can you please try downgrading to node v16 to see if you still have the `Cannot find module` error?

---

If you're still seeing license warnings on start (or it is not starting for you), please email hello@wallabyjs.com with your license key so we can investigate the issue further for you.

If you're still having problems running Quokka files in your project, it may be related to your project using ESM and breaking changes that have been made in `node.js`. You can confirm if this is the case by using the latest version of Quokka and switching to community edition (do not use PRO features, and confirm that Quokka works for your file).

**Comment by greqov** [comment]
> Can you please try the `quokka.suppressExpirationNotifications` setting and see if the warning disappears?

Yep, it disappears.

> Can you please try downgrading to node v16 to see if you still have the Cannot find module error?

I installed nvm, installed and switched to node v16. But even then quokka uses the latest node version (and outputs an error). I don't where it come from.
```
​​​​​Quokka PRO 'Untitled-1.js' (node: v21.6.1)​​​​
```
Is there a way to tell quokka to use specific node version?

I tried this extension on another PC (Windows 10, without any nodejs installed). It works, it uses node 18.17.1, the same version that we can see in VSCode 'About' window. So _technically_ I can use that old purchased version of extension (probably until installing newer nodejs).

So it seems like in ideal scenario I can choose between:
- old nodejs + quokka + sacrificed other software;
- latest nodejs + all nodejs-related software but without PRO quokka.

I have to admit you've "tricked" me with this purchase. I thought that I buy extension with certain list of features and I can use it "forever". In reality I have to hassle with outdated software to make it work.
